src/transformers/models/grounding_dino/processing_grounding_dino.py

```diff
@@ -56,6 +64,26 @@ def get_phrases_from_posmap(posmaps, input_ids):
     return token_ids
 
 
+class GroundingDinoProcessorKwargs(ProcessingKwargs, total=False):
+    _defaults = {
+        "text_kwargs": {
+            "add_special_tokens": True,
+            "padding": False,
+            "truncation": None,
+            "max_length": None,
+            "stride": 0,
+            "pad_to_multiple_of": None,
+            "return_attention_mask": None,
+            "return_overflowing_tokens": False,
+            "return_special_tokens_mask": False,
+            "return_offsets_mapping": False,
+            "return_token_type_ids": True,
+            "return_length": False,
+            "verbose": True,
+        }
+    }
+
```

Comment:
I think we're supposed to add default image kwargs here as well, but I also didn't see default image kwargs on the merged AlignProcessor. @molbap what is the correct way?

Reply:
@zucchini-nlp I was also hesitated to write like this. I will change this as he comments!

Reply:
Yes exactly, this should include all defaults for all modalities. However here, the base processor was 
1) not passing any of the kwargs to the `image_processor` 
2) hence, not passing any defaults to the same image processor.
So, we should not pass default image kwargs here, but simply make sure image processing kwargs are received by the image processor post `self._merge_kwargs` as done now.

Reply:
ah, just saw @EduardoPach comment above - it seems `annotations` and `do_convert_annotations` should have been passed to the image processor but weren't. So, they don't have currently any default value, the point above does not change

Reply:
@zucchini-nlp @molbap Also now I have noticed that current PR might be having conflict from [PR](https://github.com/huggingface/transformers/pull/31368/files#diff-6505546ec5a9ab74b2ce6511681dd31194eb91e9fa3ce26282e487a5e61f9356). So I will pull his branch whenever it is ready to merge